crates/cargo-remove/tests/testsuite/cargo_remove/invalid_remove_target/stderr.log

```diff
@@ -1,2 +1,2 @@
     Removing dbus from dependencies for target `powerpc-unknown-linux-gnu`
-error: The table `powerpc-unknown-linux-gnu` could not be found.
+error: The dependency `dbus` could not be found in `target.powerpc-unknown-linux-gnu.dependencies`.
```

Comment:
I'm confused by this commit as there is no corresponding source change.

Reply:
Yes, I'm also confused. This was almost certainly introduced in ec94271/8fbd033, but for some reason in both cases the tests passed. After I started working on something else, the test failure came up. Maybe something's being cached/skipped somehow...?

Reply:
Ah, I see what happened. CI/CD `cargo test` needs a `--all` flag. I've been careful to make sure that my commits pass all tests (with one exception, ec94271), so there's not much of an issue there for older commits. I can update `build.yml` on this branch to fix this.

Reply:
Yes, please do so (I think I did the same thing in merge-add now that you mention it).  I just universally pass --workspace in my standard Ci template but I decided to not carry my template over into this project but kept it was I found it.